bpf/lib/lb.h

```diff
@@ -47,6 +47,17 @@ struct bpf_elf_map __section_maps LB6_BACKEND_MAP = {
 	.flags          = CONDITIONAL_PREALLOC,
 };
 
+#ifdef ENABLE_SESSION_AFFINITY
+struct bpf_elf_map __section_maps LB6_AFFINITY_MAP = {
+	.type		= BPF_MAP_TYPE_LRU_HASH,
+	.size_key	= sizeof(struct lb6_affinity_key),
+	.size_value	= sizeof(struct lb_affinity_val),
+	.pinning	= PIN_GLOBAL_NS,
+	.max_elem	= CILIUM_LB_MAP_MAX_ENTRIES,
```

Comment:
Probably worth reconsidering?

Reply:
Hmm, so given this is hit from outside by also inside the cluster it's sort of in similar situation as with NAT map. Maybe both sizes should be similar then?

Reply:
Commit msg is obsolete btw (`This commit introduces two (in the future with ipv6 - three) ...`)?

Reply:
> sort of in similar situation as with NAT map. Maybe both sizes should be similar then?

Hmm, but NAT maps are not used when accessed from inside the cluster.